src/region/RegionAnnotations.tsx

```diff
@@ -146,7 +162,8 @@ export default class RegionAnnotations extends React.Component<Props, State> {
                 {isCreating && staged && canReply && targetRef && (
                     <div className="ba-RegionAnnotations-popup">
                         <PopupReply
-                            defaultValue={staged.message}
+                            editorState={editorState}
+                            intl={intl}
```

Comment:
The PopupReply component can access `intl` from context rather than having it passed in as a prop from the parent.